client/state/posts/selectors.js

```diff
@@ -359,6 +363,31 @@ export const isEditedPostDirty = createSelector(
 );
 
 /**
+  * Returns true if the edited post has content
+  * (title, excerpt or content not empty)
+  *
+  * @param  {Object}  state  Global state tree
+  * @param  {Number}  siteId Site ID
+  * @param  {Number}  postId Post ID
+  * @return {Boolean}        Whether the edited post has content or not
+ */
+export function editedPostHasContent( state, siteId, postId ) {
```

Comment:
Ideally the `has` (or `is`, or `get`) would come first in the selector name as an indicator of the type of value it returns... but I'm struggling to come up with a better name 😄